e-learning-webapps/package.json

```diff
@@ -49,7 +49,9 @@
     "webpack-dev-server": "^3.1.4",
     "webpack-merge": "^4.1.3"
   },
-  "dependencies": {},
+  "dependencies": {
+    "axios": "^0.21.1"
+  },
```

Comment:
We should use JS Fetch API , do not use Axios

Reply:
Understood.